### ISSUE TITLE
Lock models enhancement

### ISSUE THREAD

**Issue by ebuechner** [smeighan/xLights#1379]
I'm looking for the ability to lock models together in the layout screen in some way.
For instance an AC tree made up of different colored strands that needs to be layered could have all the different model layers locked together. 
Or have a mega tree and star locked together, so moving and resizing would work in unison.

**Comment by AzGilrock** [comment]
I don't think we need to add any features to accomplish this.  If you put them all in a model group just click the model group name on the left which selects them all then hold down control key and click and drag in the window.

**Comment by ebuechner** [comment]
Thanks, I didn't think of using the model group as a method for group selecting models in the layout window. 
Could we at least get a right click option while all the models are selected to lock all or unlock all?
